src/rust/src/orchard_bundle.rs

```diff
@@ -66,29 +138,48 @@ impl Bundle {
         self.inner().map(|b| b.actions().len()).unwrap_or(0)
     }
 
+    /// Returns whether the Orchard bundle is present and spends are enabled.
```

Comment:
Documentation is wrong.

Reply:
(This is for `enable_spends`)

Reply:
I'm confused: this _is_ documenting `enable_spends` (I haven't changed the branch commits since your review).